x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_container.tsx

```diff
@@ -68,14 +68,22 @@ export const LatestFindingsContainer = ({ dataView }: FindingsBaseProps) => {
    */
   const findingsGroupByNone = useLatestFindings({
     ...getPaginationQuery({
-      pageIndex: urlQuery.pageIndex,
-      pageSize: pageSize || urlQuery.pageSize,
+      pageIndex: 0,
+      pageSize: MAX_ITEMS,
     }),
     query: baseEsQuery.query,
     sort: urlQuery.sort,
     enabled: !baseEsQuery.error,
   });
 
+  const slicedPage = useMemo(() => {
```

Comment:
nit: can this piece of logic be shared? I do see that it uses `findingsGroupByNone`, I'm thinking if it can be passed as an argument to that util

something like (pseudo code to illustrate what I mean)

```
useMemo(() => {
   return getSlicedPage(pageSize, pageIndex, findingsGroupByNone ....)
})
```